.travis.yml

```diff
@@ -28,6 +29,15 @@ cache:
   - $HOME/.cache/electron
   - $HOME/.cache/electron-builder
 
+before_script:
+  # Replace clientId and clientSecret values in app-config.js with environment
+  # variable values.
+  - cd eapp/config/
+  - cp ref-config.js app-config.js
```

Comment:
copying ref-config.js to app-config.js  is no longer needed. app-config.js has all the directories list and the variables names and is being tracked by git. Earlier it was in gitignore, so that step was needed